src/manager/node.c

```diff
@@ -930,6 +930,25 @@ static int node_disconnected(UNUSED sd_bus_message *message, void *userdata, UNU
         /* Remove anonymous nodes when they disconnect */
         if (node->name == NULL) {
                 manager_remove_node(manager, node);
+        } else {
+                /* Remove all jobs associated with the registered node that got
+                   disconnected. */
+                if (!LIST_IS_EMPTY(manager->jobs)) {
+                        Job *job = NULL;
+                        LIST_FOREACH(jobs, job, manager->jobs) {
+                                if (strcmp(job->node->name, node->name) == 0) {
+                                        hirte_log_debugf(
```

Comment:
This log seems off. `hirte_log_*` takes care of getting the file/line/func info. I think the first 3 parameters need to be removed.

Reply:
I will remove file/line/func info for purposes of this PR, and I'll find out why I'm not getting those in my print output.

Reply:
@engelmi solved this issue for me.  I needed to use "stderr-full" in my hirte config in order to get the file/line/func info my prints.  Thanks Michael!